State.py

```diff
@@ -261,11 +263,11 @@ def can_pass_boulder(self, boulder: str, age: str, **kwargs) -> bool:
 
     def can_pass_boulder_type(self, boulder_type:BOULDER_TYPE, age: str, **kwargs) -> bool:
         if boulder_type == BOULDER_TYPE.BRONZE:
-            # Check for hammer and adult
-            return age == 'adult' and self.has(Megaton_Hammer)
+            # Check for hammer and adult or the ability to equip swap
+            return self.has(Megaton_Hammer) and (age == 'adult' or self.can_equip_swap(self, age='child'))
         elif boulder_type == BOULDER_TYPE.SILVER:
             # Check for adult+str2
-            return age == 'adult' and self.has(Progressive_Strength_Upgrade, 2)
+            return (age == 'adult' and self.has(Progressive_Strength_Upgrade, 2))  
```

Comment:
Please run the CI script to remove things like trailing whitespace

`python3 CI.py --fix --no_unit_tests`